.github/workflows/ci_dockers.yml

```
@@ -93,12 +93,12 @@ jobs:
       matrix:
         # the config used in '.github/workflows/ci_test-conda.yml'
         python_version: ["3.8"]
-        pytorch_version: ["1.6", "1.7", "1.8", "1.9", "1.10"]
+        pytorch_version: ["1.7.1", "1.8", "1.9", "1.10"]
```

```suggestion
        pytorch_version: ["1.7", "1.8", "1.9", "1.10"]
```
That should work since we run 3.8 here, no?